airflow/ui/src/pages/Dag/Overview/Overview.tsx

```diff
@@ -29,7 +29,7 @@ import TimeRangeSelector from "src/components/TimeRangeSelector";
 import { TrendCountButton } from "src/components/TrendCountButton";
 import { stateColor } from "src/utils/stateColor";
 
-const defaultHour = "12";
+const defaultHour = "168";
```

Comment:
This could be heavy on the database being default when there are lot of entries in the database. IIRC the cluster activity page was also slower on large Airflow instances with higher time interval.

Reply:
Yeah, this is basically for local development. I think long-term this might become a config option.